optuna_integration/xgboost.py

```diff
@@ -0,0 +1,124 @@
+from typing import Any
+
+import optuna
+import optuna_integration
```

Comment:
```suggestion
from typing import Any

import optuna

import optuna_integration
```

Reply:
... and this change to the imports is done as well!